src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs

```diff
@@ -1490,7 +1496,11 @@ int IComparable.CompareTo(object? obj)
                 return this.CompareTo(other);
             }
 
+#if SYSTEM_PRIVATE_CORLIB
```

Comment:
Why is this corelib-specific? The Microsoft.Bcl.Memory package can't include the same resource?

Reply:
`Microsoft.Bcl.Memory` does not include any resources at present. And the non-corelib path is highly unlikely to be actually hit.